tests/nativeapp/test_artifacts.py

```diff
@@ -36,6 +36,10 @@
 
 from tests.nativeapp.utils import assert_dir_snapshot, touch
 from tests.testing_utils.files_and_dirs import pushd, temp_local_dir
+from tests_common import IS_WINDOWS
+
+if IS_WINDOWS:
+    pytest.skip("Does not work on Windows", allow_module_level=True)
```

Comment:
Skipped due too many tests using hardcocded paths.

Reply:
Not following what you mean, can you explain why this is failing on windows? Is it because some paths are absolute in the snapshots?